README.md

```diff
@@ -126,39 +131,27 @@ By design Services should never expose DB Entities outside. All communication wi
 [Mapstruct](https://mapstruct.org/) is advised to use for mapping Entities to DTOs and vice versa.
 
 Typical Service & DTO pattern:
-```java
-    @Override
-    public ExampleEntityFirstDto save(ExampleEntityFirstDto exampleEntityFirstDto) {
-        ExampleEntityFirst exampleEntityFirst = exampleEntityFirstMapper.toEntity(exampleEntityFirstDto);
-        exampleEntityFirst = exampleEntityFirstRepository.save(exampleEntityFirst);
-        return exampleEntityFirstMapper.toDto(exampleEntityFirst);
-    }
-```
-
-### CI/CD
-
-[//]: # (TODO: define travice/gitlab file)
-
-### Actuator
-
-#### Healthcheck
-
-[//]: # (TODO: need to find helathckeck for Kafka)
 
-#### Metrics
-Metrics should be collected in Prometheus format using Micrometer.
-You can access metric endpoint by `http://localhost:8081/management/prometheus`
-
-#### Log management
-Endpoint for logs management `http://localhost:8081/management/logs`
+```java
+@Override
+public ExampleEntityFirstDto save(ExampleEntityFirstDto exampleEntityFirstDto) {
+  ExampleEntityFirst exampleEntityFirst = exampleEntityFirstMapper.toEntity(exampleEntityFirstDto);
+  exampleEntityFirst = exampleEntityFirstRepository.save(exampleEntityFirst);
+  return exampleEntityFirstMapper.toDto(exampleEntityFirst);
+}
 
-#### Introspections
+```
 
-Endpoint for thread dump: `http://localhost:8081/management/threaddump`
+In the project root, JHipster generates configuration files for tools like git, prettier, eslint, husky, and others that are well known and you can find references in the web.
 
-### Error handling
+`/src/*` structure follows default Java structure.
 
-Commons `xm-commons-i18n` contains `com.icthh.xm.commons.i18n.error.web.ExceptionTranslator` class which is responsible for exception translation.
+- `.yo-rc.json` - Yeoman configuration file
```

Comment:
seems these jhipster-specific files were removed. should we keep info about them here?

Reply:
README.md is not updated yet
ignore it for now

Reply:
done